contents/docs/session-replay/how-to-control-which-sessions-you-record.mdx

```diff
@@ -95,15 +76,16 @@ You can also set a minimum duration for sessions to be recorded. This is useful
     classes="rounded"
 />
 
-The minimum duration is set in seconds. Whenever a new session starts, the browser records the start time. If the minimum duration has passed since the start time, the session data is sent. If it hasn't, the session continues to be buffered in-memory. 
+This is useful if you want to exclude sessions that are too short to be useful. For example, you might want to exclude sessions that are less than 2 seconds long to avoid recording sessions where users quickly bounce off your site.
 
 ### Limitations
 
+The minimum duration is set in seconds. Whenever a new session starts, the browser records the start time. If the minimum duration has passed since the start time, the session data is sent. If it hasn't, the session continues to be buffered in-memory. 
+
 This means that if you set a high minimum duration and your user visits multiple pages each for a short time, you still record the session but miss the beginning. If the user leaves the site before the minimum duration has passed, the session is not recorded.
 
 If you find you are missing the beginning of sessions, you can reduce the minimum duration or use one of our other methods to reduce the number of sessions you record.
```

Comment:
I dont understand this section. Waiting to hear back [here](https://posthog.slack.com/archives/C03PB072FMJ/p1725442163354819) before I rewrite